DOCS-1937: Update helmet tutorial instructions and screenshots R2D2 (#2778)
viamrobotics/docs: docs/tutorials/projects/helmet.md

2. Make sure your computer (whether it's a personal computer or an SBC) is connected to adequate power, and turn it on.

> 3. If you haven't already, [create a new machine](/fleet/machines/#add-a-new-machine) in the [Viam app](https://app.viam.com) and follow the instructions to [install `viam-server`](/get-started/installation/), the software you will use to operate your machine, on the computer you're using for this project.

Reviewer: ```suggestion
3. If you haven't already, [add a new machine](/fleet/machines/#add-a-new-machine) in the [Viam app](https://app.viam.com) and follow the setup instructions to install `viam-server` on the computer you're using for this project and connect to the Viam app.
```
A bit simpler - I'll assign myself the ticket to go through all tutorials to change it to something like this at the end

1. First, you need to add the data service to your machine to make it available to capture data on your camera.

>    Navigate to tyour machine's **CONFIGURE** tab.

Author: ```suggestion
   Navigate to your machine's **CONFIGURE** tab.
```
